docs/source/GetStartedGuide.rst

```diff
@@ -93,6 +104,12 @@ Note that `python` and `pip` should point to the Python interpreter version 2.7.
 If you're having trouble to install, you can try again and use the command line
 utilities `python2.7` and `pip2.7`.
 
+Please note that some Avocado functionality may be implemented by
+optional plugins.  To install say, the HTML report plugin, run::
+
+    cd optional_plugins/html
```

Comment:
I know, but I'm talking about `install`. (I'm fine on my developer machine, but my CI machines, which are also used for manual interaction didn't have the html plugin installed)